src/main/java/ui/IdGenerator.java

```diff
@@ -19,6 +19,14 @@ public static String getRepositorySelectorId() {
         return "repositorySelector";
     }
 
+    public static String getRepositoryPickerTextFieldId() {
+        return "repositoryPickerUserInputField";
+    }
+
+    public static String getRepositoryPickerSuggestedRepoListId() {
+        return "suggestedRepositoryList";
+    }
+
```

Comment:
Ah I see, can just take care of your own methods first or can leave them as they are.